### PR TITLE
T4 articles list

### PR DESCRIPTION
**Pull request by manheicatherine**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,72 @@
...
+
+img {
+  width: 400px;
+  height: 300px;
+  border-radius: 10%;
+  object-fit: cover;
+}
+
+@media (prefers-reduced-motion: no-preference) {
 }
```
Comment (elijaharley) [inline src/App.css:72]: Some really great work here, Catherine - you've clearly paid a lot of attention to detail with styling and its really impactful.

File: src/components/Article.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <h4>
+          {article.body}
+          <br></br>
+          <br></br>
+          {article.created_at}
+        </h4>
+      </li>
+    </section>
+  );
+}
```
Comment (elijaharley) [inline src/components/Article.jsx:18]: The content here makes it difficult to provide useful alt text, in this instance I'd be inclined to give an alt of 'Generic ${article.topic} image' or similar, just to give slightly more detail.

File: src/components/ArticlesList.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <section>
+      <ol>
+        {articles.map((article) => {
+          return <Article article={article} key={article.article_id} />;
+        })}
+      </ol>
+    </section>
+  ); 
+  }
+}
```
Comment (elijaharley) [inline src/components/ArticlesList.jsx:30]: Nice use of loading state here

File: src/utils/api.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+const articlesApi = axios.create({
+  baseURL: "https://catherine-nc-news.onrender.com/api",
+});
+export function getArticles(article_id) {
+  return articlesApi
+    .get("./articles", { params: { article_id: article_id } })
+    .then(({ data }) => {
+      return data.articles;
+    });
+}
```
Comment (elijaharley) [inline src/utils/api.js:12]: Nice use of an external api file for axios requests and implementation of axios.create.

**Review by elijaharley — APPROVED** [review]
Great stuff Catherine, please see comments inline.

File: src/components/Article.jsx (hunk 2 above)
Comment (manheicatherine) [reply src/components/Article.jsx:18]: Thank you Eli for all the comments, I will change it! : )

**Pull request merged**